anyway/infographics_utils.py

```diff
@@ -278,20 +368,20 @@ def get_most_severe_accidents_table_title(location_text):
     return "תאונות חמורות ב" + location_text
```

Comment:
gettext("severe accidents in") + ..

Reply:
I do not follow. Is this related to I18N?

Reply:
yes, but you can skip that for now and we'll wrap it later